src/main/java/org/kuali/kfs/module/purap/document/service/impl/PaymentRequestServiceImpl.java

```diff
@@ -30,7 +30,7 @@
 import org.kuali.kfs.fp.FPKeyConstants;
 import org.kuali.kfs.fp.document.service.DisbursementVoucherValidationService;
 import org.kuali.kfs.kim.api.identity.Person;
-import org.kuali.kfs.krad.bo.DocumentHeader;
+import org.kuali.kfs.sys.businessobject.DocumentHeader;
```

Comment:
There appear to be two lines in this file that are importing the same DocumentHeader class. Please remove one of the duplicates.

Reply:
updated